pkg/downloader/downloader_test.go

```diff
@@ -80,3 +82,37 @@ func TestDownloadRemote(t *testing.T) {
 		assert.Equal(t, StatusUsedCache, r.Status)
 	})
 }
+
+func TestDownloadLocal(t *testing.T) {
```

Comment:
```go
if runtime.GOOS == "windows" {
  // FIXME: `TempDir RemoveAll cleanup: remove C:\users\runner\Temp\TestDownloadLocalwithout_digest2738386858\002\test-file: Sharing violation.`
  t.Skip("Skipping on windows")
}
```

Reply:
Done. I took a quick look, on how the windows test run via wine and will try figure how to fix/workaround. Sorry for the delay on this PR my end - the end of the year became very busy for me